lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
FInally... the compiler has a windows/config.h that's different than the library one...